assets/src/edit-story/components/canvas/layout.js

```diff
@@ -215,7 +217,7 @@ const PageArea = forwardRef(
   (
     {
       children,
-      showDangerZone,
+      showSafeZone,
```

Comment:
Should it have a default of `false`? Is the `PageArea` component not used more than once?